react/components/AutocompleteInput/Option.tsx

```diff
@@ -54,11 +54,11 @@ const Option: React.FunctionComponent<PropTypes.InferProps<
     const match = value.substr(index, searchTerm.length)
     const suffix = value.substring(index + match.length)
     return (
-      <span className="truncate">
-        <span className="fw7">{prefix}</span>
-        {match}
-        <span className="fw7">{suffix}</span>
-      </span>
+      <>
+        {prefix}
+        <span className="fw7">{match}</span>
+        {suffix}
+      </>
```

Comment:
How was this bold text change validated?

Reply:
I'll wait for oldufre's approval, but I think that doesnt make sense to highlight the text that doesn't match with the user input

Reply:
Well, I've done it this way because we've decided it together. A pretty good example:

![image](https://user-images.githubusercontent.com/15948386/77449708-0fddad80-6dd1-11ea-9c41-8fe8a55340a6.png)

Reply:
The autocomplete idea is not to highlight what matches. Its idea is to highlight the **suggestions**. Your logic makes perfect sense in the `Select` component.

Reply:
I agree with @estacioneto

Reply:
Oh, ok then. But in that case we only have suffixes, then its a different case, no?

Reply:
Actually not

![image](https://user-images.githubusercontent.com/15948386/77450459-06a11080-6dd2-11ea-943a-2743a3c9df15.png)

Reply:
[Kinda OFF] Related to the `admin-collections`... Did you guys notice that your autocomplete filter is not working?

Reply:
Ok, I've reverted the bold

Reply:
I still think is weird because most of suggestions in these cases are in the middle, but it may be just effect of the products names... About the filter I've managet to filter after pressing enter, but maybe would be good if the results were filtered as the user types

Reply:
Totally. The filter cannot happen when pressing enter. This behavior should be related to the user's ability search by term.

Reply:
But anyway, I still feel weird in our case because is not unusual to have options that don't have a matching text in results while we're waiting for results... But I don't know, may be just a biased perception

Reply:
You can filter the current results when typing. I remember we just did it that way because it's a paged resource and we didn't want to add another filter logic when, in some moment, the GraphQL API would do it.